experimental/README.md

```diff
@@ -42,6 +42,23 @@ Unlike the regular Docker binary, the experimental channels is built and updated
 
     This command downloads a test image and runs it in a container.
 
+### Get the Linux binary
+To download the latest experimental `docker` binary for Linux,
+use the following URLs:
+
+    https://experimental.docker.com/builds/Linux/i386/docker-latest
+
+    https://experimental.docker.com/builds/Linux/x86_64/docker-latest
+
+After downloading the appropriate binary, you can follow the instructions
+[here](https://docs.docker.com/installation/binaries/#get-the-docker-binary) to run the `docker` daemon.
+
+> **Note**
```

Comment:
Sorry for that, I suggested @ankushagarwal to copy it from the existing docs https://docs.docker.com/installation/binaries/#get-the-docker-binary, which was formatted this way, so thought it would be okay